packages/suite/src/hooks/wallet/trading/form/useTradingBuyForm.ts

```diff
@@ -112,10 +112,7 @@ export const useTradingBuyForm = ({
     const draftUpdated: TradingBuyFormProps | null = draft
         ? {
               ...draft,
-              fiatInput:
-                  draft.fiatInput && draft.fiatInput !== ''
-                      ? draft.fiatInput
-                      : buyInfo?.buyInfo?.defaultAmountsOfFiatCurrencies.get(suggestedFiatCurrency),
+              fiatInput: draft.fiatInput && draft.fiatInput !== '' ? draft.fiatInput : undefined,
```

Comment:
nit: this might make it even simplier?
```suggestion
              fiatInput: draft?.fiatInput !== '' ? draft.fiatInput : undefined,
```

Reply:
Not sure whether it matters here, but suggested change behaves differently.

```ts
type Draft = { fiatInput: string | undefined | null };

describe('Actually not same', () => {
    const fnA = (draft: Draft) =>
        draft.fiatInput && draft.fiatInput !== '' ? draft.fiatInput : undefined;

    const fnB = (draft: Draft) => (draft?.fiatInput !== '' ? draft.fiatInput : undefined);

    describe.each([
        ['fnA', fnA],
        ['fnB', fnB],
    ])('for %s', (_, testFn) => {
        it.each<[Draft, string | undefined]>([
            [{ fiatInput: '100' }, '100'],
            [{ fiatInput: '' }, undefined],
            [{ fiatInput: undefined }, undefined],
            [{ fiatInput: null }, undefined],
        ])('should return correct value for input %o', (input, expected) => {
            expect(testFn(input)).toBe(expected);
        });
    });
});
```

Reply:
`fnA` checks for truthiness, while `fnB` only checks for inequality against the empty string. Therefore `fnB` returns `null` for `fiatInput: null`. I'd personally keep the original logic for consistency.